src/_listObject.scss

```diff
@@ -0,0 +1,228 @@
+
+@import "./listDefinitions.scss";
```

Comment:
plz remove `.scss` it is not needed

Reply:
styles are not getting loaded sometimes when I am removing of .scss.